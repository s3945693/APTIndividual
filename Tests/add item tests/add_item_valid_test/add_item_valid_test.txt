1
4
Chicken Burger
Yummy burger with chicken, mayo and lettuce
4.50
1
9