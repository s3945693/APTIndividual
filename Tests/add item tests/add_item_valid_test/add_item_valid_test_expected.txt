
Main Menu
    1. Display Items
    2. Purchase Items
    3. Save and Exit
Administrator-Only Menu:
    4. Add Item
    5. Remove Item
    6. Display Coins
    7. Reset Stock
    8. Reset Coins
    9. Abort Program

Select your option (1-9): Items Menu
----------
ID   |Name                                    | Available | Price  
-------------------------------------------------------------------
I0002|Apple Pie                               |20         |$ 3.00
I0003|Lemon Cheesecake                        |10         |$ 4.00
I0004|Lemon Meringue Pie                      |20         |$ 3.00
I0005|Lemon Tart                              |12         |$ 3.75
I0001|Meat Pie                                |50         |$ 3.50

Main Menu
    1. Display Items
    2. Purchase Items
    3. Save and Exit
Administrator-Only Menu:
    4. Add Item
    5. Remove Item
    6. Display Coins
    7. Reset Stock
    8. Reset Coins
    9. Abort Program

Select your option (1-9): The id for the new item will be I0006
Enter the name of the item: Enter the description of the item: Enter the price of the item: This item "Chicken Burger - Yummy burger with chicken, mayo and lettuce" has now been added to the menu.

Main Menu
    1. Display Items
    2. Purchase Items
    3. Save and Exit
Administrator-Only Menu:
    4. Add Item
    5. Remove Item
    6. Display Coins
    7. Reset Stock
    8. Reset Coins
    9. Abort Program

Select your option (1-9): Items Menu
----------
ID   |Name                                    | Available | Price  
-------------------------------------------------------------------
I0002|Apple Pie                               |20         |$ 3.00
I0006|Chicken Burger                          |20         |$ 4.50
I0003|Lemon Cheesecake                        |10         |$ 4.00
I0004|Lemon Meringue Pie                      |20         |$ 3.00
I0005|Lemon Tart                              |12         |$ 3.75
I0001|Meat Pie                                |50         |$ 3.50

Main Menu
    1. Display Items
    2. Purchase Items
    3. Save and Exit
Administrator-Only Menu:
    4. Add Item
    5. Remove Item
    6. Display Coins
    7. Reset Stock
    8. Reset Coins
    9. Abort Program

Select your option (1-9): 
Goodbye!

